americangut/question/ag_question.py

```diff
@@ -0,0 +1,117 @@
+import inspect
+
+import numpy as np
+
+
+class AgQuestion:
+    """A base object class for handling American Gut Data dictionary entries"""
+    true_values = {'yes', 'y', 'true', 1, 1.0, 'Y', 'YES', 'Yes', True}
+    false_values = {'no', 'n', 'false', 0, 0.0, 'N', 'NO', 'No', False}
+
+    def __init__(self, name, description, dtype, clean_name=None, **kwargs):
+        """A base object for describing single question outputs
+
+        Parameters
+        ----------
+        name : str
+            The name of column where the group is stored
+        description : str
+            A brief description of the data contained in the question
+        dtype : object
+            The datatype in which the responses should be represented. (i.e.
+            float, int, str).
+        clean_name : str, optional
+            A nicer version of the way the column should be named.
+        remap : function
+            A function used to remap the data in the question.
+        free_response: bool, optional
+            Whether the question is a free response question or controlled
+            vocabulary
+        mimmarks : bool, optional
```

Comment:
i see below about the inquiry into kwargs, why not just setup defaults in the definition?